src/main/java/org/terasology/launcher/util/DownloadUtils.java

```diff
@@ -31,6 +31,7 @@
 import java.io.BufferedInputStream;
 import java.io.BufferedOutputStream;
 import java.io.BufferedReader;
+import java.io.FileNotFoundException;
```

Comment:
🧐 is this used? how could this import been missing before...

Reply:
right you are :+1: